app/components/forms/photo-album-form.hbs

```diff
@@ -73,9 +73,9 @@
         @errors={{@model.errors}}
         @errorMessage={{@errorMessage}}
       >
-        <LinkTo @route='photo-albums.index' class='btn btn-default'>
+        <button {{action @onCancel}} type="button" class='btn btn-default'>
```

Comment:
This appears to be an issue for _all_ cases where this sequence of actions is possible. I've discovered why this is, but the fix isn't entirely straightforward. I think it is because the controller has a `successTransitionModel` that is not up to date with its `model`. I suspect this is because modifying the `model` of the controller in the following way:
`this.model = ...something...` doesn't _change_ anything about the object, but rather replaces it, meaning it doesn't trigger reactivity.

Reply:
Hmm, then that is the only one of the variables that is a function so it will work but it makes it less consistent. Could you not override Route.setupController in the NewController or DestroyController

Reply:
I think that is not quite the solution, as the cancelTransitionModel and successTransitionModel both suffer this downside, and they are not always pointing to the model object. We could try using getters for both properties?
Then we can still sort of use them as properties, while they are also reactive to changes.
for example:
```javascript
get successTransitionModel() {
    return this.model;
}
```
would also detect when this.model changes in its entirety

Reply:
That looks like a nice solution

Reply:
I'll implement it tomorrow, then we can finally get this PR merged

Reply:
done